Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/ParameterResolution.dfy.expect

```diff
@@ -45,7 +45,8 @@ ParameterResolution.dfy(163,13): Error: a refining formal parameter ('x') in a r
 ParameterResolution.dfy(172,14): Error: a refining formal parameter ('x') in a refinement module is not allowed to give a default-value expression
 ParameterResolution.dfy(174,12): Error: a refining formal parameter ('x') in a refinement module is not allowed to give a default-value expression
 ParameterResolution.dfy(169,12): Error: wrong number of arguments (got 0, but function 'O' expects 1: (x: int))
-ParameterResolution.dfy[RefinementB](147,12): Error: wrong number of arguments (got 0, but function 'O' expects 1: (x: int))
+ParameterResolution.dfy(147,12): Error: wrong number of arguments (got 0, but function 'O' expects 1: (x: int))
```

Comment:
Wow I did not expect refinement to make error reporting so difficult. I'm a bit puzzled that main errors are on the refined module and related location on the refining module, when it's the refining module which has an error. But I'm not expecting to change this right now since it's the current behavior.